src/content/enterprise/install/releases.mdx

```diff
@@ -31,6 +31,7 @@ May 19, 2022
 
 ### Breaking changes
 - Ingresses using the `kubernetes.io/ingress.class` annotation will be automatically updated by the okteto webhook and saved as `spec.ingressClassName` with the annotation removed
+- Okteto Pipelines are executed relative to the path where manifest is located.
```

Comment:
`are executed relative to the path` -> @rberrelleza @RinkiyaKeDad do you find this clear enough? I think we should mention the implications, like any relative path in the manifest refers to the path where the manifest is located or something like that

Reply:
ok, i've added more context to the breaking change. let me know your thoughts thnks

Reply:
Looks good to me now after the change!